Tests/UnitTests/FoundationExtensions/NSData+RCExtensionsTests.swift

```diff
@@ -29,14 +29,29 @@ class NSDataExtensionsTests: TestCase {
     }
 
     func testAsFetchToken() {
-        let receiptFilename = "base64EncodedReceiptSampleForDataExtension"
-        let storedReceiptText = NSDataExtensionsTests.readFile(named: receiptFilename)
-        let storedReceiptData = NSDataExtensionsTests.sampleReceiptData(receiptName: receiptFilename)
+        let storedReceiptText = NSDataExtensionsTests.readFile(named: Self.receiptFilename)
+        let storedReceiptData = NSDataExtensionsTests.sampleReceiptData(receiptName: Self.receiptFilename)
         let fetchToken = storedReceiptData.asFetchToken
 
         expect(fetchToken).to(equal(storedReceiptText))
         expect(storedReceiptData.asFetchToken).to(equal(storedReceiptText))
     }
+
+    func testStringDataAsHashString() {
+        expect("sample string".asData.hashString) == "99ad9154f94977dd8913f3b7ea14091d00e52b8931c2bc1cfc7ea62b7c26727b"
+    }
+
+    func testDataAsHashStringHashesTheEntireData() {
```

Comment:
This test failed with the original `UUID` implementation, which only hashed the first 16 bytes (see https://opensource.apple.com/source/CF/CF-855.17/CFUUID.c.auto.html).